.travis.yml

```diff
@@ -16,6 +16,38 @@ script:
 # Docker Run to Execute Unit E2E Selenium Package
 - echo "Running E2E Tests"
 - docker run -e APP_KEY -e APP_TOKEN -e board_id -e list_id_Done -e list_id_Pending -e list_id_ToDo -e SECRET_KEY todo-app:test Test/selenium_test.py
+- echo "All Tests Have Been Completed"
+
+# DockerHubLogin
+- docker login --username "$docker_login" --password "$docker_pass"
+
+# Docker Build the Prod and DockerPush
+- docker build --target production --tag todo-app:prod .
+- docker tag todo-app:prod legendarypan/todo-app:prod
+- docker push legendarypan/todo-app:prod
```

Comment:
Do you really want every branch to update the `legendarypan/todo-app:prod` image on Docker Hub?

Or should this be part of the deployment?